server/tests/src/test/java/org/infinispan/server/persistence/JdbcStringBasedCacheStoreIT.java

```diff
@@ -60,6 +60,8 @@ public void testFailover(Database database) throws Exception {
             cache.remove("k1");
             assertNull(table.getValueByKey("k1"));
             cache.clear();
+        } finally {
+            SERVERS.hotrod().createRemoteCacheManager().administration().removeCache(cache.getName());
```

Comment:
@ryanemerson cache name for the tests is generated by some kind of code which is deterministic, and the resulting name is always the same. If we are running against external Database (maybe even container?) and test fails, the rerun will fail on existing data in the cache.

This isn't bullet prove solution as cache is created outside of the try catch block)